### ISSUE TITLE
DietPi issue

### ISSUE THREAD

**Issue by tester1969** [project-owner/PeppyMeter#4]
dietpi@DietPi:~/PeppyMeter2$ python3 peppymeter.py
pygame 2.0.3 (SDL 2.0.14, Python 3.9.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/home/dietpi/PeppyMeter2/peppymeter.py", line 294, in <module>
    pm.start_display_output()
  File "/home/dietpi/PeppyMeter2/peppymeter.py", line 208, in start_display_output
    self.meter.start()
  File "/home/dietpi/PeppyMeter2/vumeter.py", line 94, in start
    self.meter = self.get_meter()
  File "/home/dietpi/PeppyMeter2/vumeter.py", line 80, in get_meter
    m = factory.create_meter()
  File "/home/dietpi/PeppyMeter2/meterfactory.py", line 60, in create_meter
    return self.create_circular_meter(meter_name)
  File "/home/dietpi/PeppyMeter2/meterfactory.py", line 124, in create_circular_meter
    meter.add_background(config[BGR_FILENAME], config[METER_X], config[METER_Y])
  File "/home/dietpi/PeppyMeter2/meter.py", line 77, in add_background
    self.meter_bounding_box = img[1].get_rect()
TypeError: 'NoneType' object is not subscriptable

Trying to run on DietPI with Xorg -> XLDE -> lightgdm and get above error... Don't know where to look. All other X applications run from command line when I setup DISPLAY

**Comment by project-owner** [comment]
Hi, I've never tried to use Pygame 2 with PeppyMeter. It looks like it cannot load images. To find out why let's enable logging. Set this property to True:
https://github.com/project-owner/PeppyMeter/blob/914ebcabe037df5854ad2ac0939084b816857c5b/config.txt#L13
then add these lines:
```
        except Exception as e:
            logging.debug(e)
```
instead of these:
https://github.com/project-owner/PeppyMeter/blob/914ebcabe037df5854ad2ac0939084b816857c5b/meterutil.py#L47
run the program. check what will be printed to the console and please let me know the result.

Best regards

**Comment by tester1969** [comment]
Traceback (most recent call last):
  File "/home/dietpi/PeppyMeter/meterutil.py", line 46, in load_pygame_image
    image = pygame.image.load(path).convert_alpha()
pygame.error: File is not a Windows BMP file

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dietpi/PeppyMeter/peppymeter.py", line 294, in <module>
    pm.start_display_output()
  File "/home/dietpi/PeppyMeter/peppymeter.py", line 208, in start_display_output
    self.meter.start()
  File "/home/dietpi/PeppyMeter/vumeter.py", line 94, in start
    self.meter = self.get_meter()
  File "/home/dietpi/PeppyMeter/vumeter.py", line 80, in get_meter
    m = factory.create_meter()
  File "/home/dietpi/PeppyMeter/meterfactory.py", line 60, in create_meter
    return self.create_circular_meter(meter_name)
  File "/home/dietpi/PeppyMeter/meterfactory.py", line 124, in create_circular_meter
    meter.add_background(config[BGR_FILENAME], config[METER_X], config[METER_Y])
  File "/home/dietpi/PeppyMeter/meter.py", line 74, in add_background
    img = self.load_image(image_name)
  File "/home/dietpi/PeppyMeter/meter.py", line 111, in load_image
    return self.util.load_pygame_image(path)
  File "/home/dietpi/PeppyMeter/meterutil.py", line 48, in load_pygame_image
    logging.debug(e)
NameError: name 'logging' is not defined

**Comment by tester1969** [comment]
Hmmm  I think I have installed pygame both from pip and apt-get. Should I remove one of them?
dietpi@DietPi:~/PeppyMeter$ pip3 list
Package            Version
------------------ ---------
certifi            2021.10.8
charset-normalizer 2.0.7
idna               3.3
numpy              1.19.5
pip                20.3.4
pygame             2.0.3
PySDL2             0.9.9
requests           2.26.0
RPi.GPIO           0.7.0
setuptools         52.0.0
urllib3            1.26.7
wheel              0.34.2
dietpi@DietPi:~/PeppyMeter$ sudo apt list |grep pygame

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

python-pygame-doc/stable 1.9.6+dfsg-4 all
python3-pygame-sdl2/stable 7.4.2-1 armhf
python3-pygame/stable,now 1.9.6+dfsg-4+b1 armhf [installed]
dietpi@DietPi:~/PeppyMeter$

P.S. I unistalled version 2 and it works. BTW if you want me to debug v2 and make this work in v2 let me know and I will do
